src/js/components/Box/StyledBox.js

```diff
@@ -146,7 +146,9 @@ const justifyStyle = css`
   justify-content: ${props => JUSTIFY_MAP[props.justify]};
 `;
 
-const wrapStyle = 'flex-wrap: wrap;';
+const wrapStyle = css`
+  flex-wrap: ${props => (props.wrapProp === true ? 'wrap' : 'wrap-reverse')};
```

Comment:
I don't think this logic is correct here. If `wrapProp` is false, we will be setting `flex-wrap: wrap-reverse`.

Reply:
hm, not sure. i think it would not pass this https://github.com/grommet/grommet/blob/03771dda4802b61fe943567c804f926df3c4bcda/src/js/components/Box/StyledBox.js#L577

Reply:
I see.

If the user passes 'garbage', then we will also set `wrap-reverse`. I think we could be a bit more defensive and only set `wrap` if `true` and `wrap-reverse` if `reverse`.

Reply:
and what should we do when user passes `garbage`? normally user will see a warning from the proptypes so i am not sure this is really needed but even i am not sure what the best course of action is here. if garbage then nowrap?

Reply:
Yes, I would think `garbage` is the same as `undefined`.